presto-main/src/main/java/com/facebook/presto/testing/TestingMetadata.java

```diff
@@ -173,6 +173,19 @@ public void createTable(ConnectorSession session, ConnectorTableMetadata tableMe
     }
 
     @Override
+    public void createTable(ConnectorSession session, ConnectorTableMetadata tableMetadata, boolean failIfExists)
+    {
+        try {
+            createTable(session, tableMetadata);
+        }
+        catch (IllegalArgumentException e) {
```

Comment:
This very generic exception often indicates a bug due to invariant violations, so we should avoid catching it. Can we make this a more specific exception?